bitnami/elasticsearch/values.yaml

```diff
@@ -208,6 +208,10 @@ image:
   ##
   debug: false
 
+## @param architecture Elasticsearch architecture (`standalone` or `cluster`)
+##
+architecture: cluster
```

Comment:
What if we add some extra context about the implications of setting `standalone` here?
- Only the `master` role configuration would be taken into account, the rest will be ignored: `data`, `coordinating`...
- `.master.replicaCount` will be ignored
- ...